### PR TITLE
jom: add mirror

### PR DESCRIPTION
**Pull request by ChrisCatCP**

### CODE REVIEW

File: packages/j/jom/xmake.lua [hunk 1]
```diff
@@ -4,9 +4,10 @@ package("jom")
     set_description("the parallel make tool for Windows.")
     set_license("GPL")
 
-    add_urls("https://download.qt.io/official_releases/jom/jom_$(version).zip", {version = function (version)
-        return version:gsub("%.", "_")
-    end})
+    add_urls("https://download.qt.io/official_releases/jom/jom_$(version).zip",
+    "https://mirrors.ocf.berkeley.edu/qt/official_releases/jom/jom_$(version).zip",
+    "https://mirrors.ukfast.co.uk/sites/qt.io/official_releases/jom/jom_$(version).zip",
```
Comment (c8ef) [inline packages/j/jom/xmake.lua:9]: I'm not sure if it's a good idea for package managers to handle the 'proxy' job manually for users. Instead, you could make this package available in _xmake-mirror/build-artifacts_. This way, as long as the user can access GitHub, they can access the relevant package.

Comment (ChrisCatCP) [reply packages/j/jom/xmake.lua:9]: only add mirror

Comment (ChrisCatCP) [reply packages/j/jom/xmake.lua:9]: I have already provided this suggestion to ruki

Comment (c8ef) [reply packages/j/jom/xmake.lua:9]: Understood. Not blocking, but that would be my take on this.

**Pull request merged**